pkg/util/network.go

```diff
@@ -43,7 +43,7 @@ func DownloadFile(destPath string, fileURL string, progressBar bool, shaSumURL s
 	}
 	client.Backoff = retryablehttp.DefaultBackoff
 	client.Logger = nil
-	client.HTTPClient.Timeout = 20 * time.Second
+	client.HTTPClient.Timeout = 5 * time.Minute
```

Comment:
Is this change only for downloading images on upgrade? If not, what other scenarios use this timeout?

Reply:
Has nothing to do with images, only things that use DownloadFile(), which means docker-compose, mutagen, and `ddev add-on get` downloads at this point.